meson.build

```diff
@@ -157,6 +157,33 @@ if init_system == 'systemd'
     install_mode: 'rw-r--r--',
     install_tag: 'systemd',
   )
+elif init_system == 'sysvinit_openrc'
+  openrc = dependency('openrc')
+  udev = dependency('udev')
+  openrc_initd_dir = '/etc/init.d'
```

Comment:
nitpick: This variable is only used once. The code will be simpler if no variable is assigned and we just use a string literal on line 174 instead.